docs/competing_risks.md

```diff
@@ -0,0 +1,278 @@
+
+.. math::
+   Pr(D_i=d_i|T_i= t,X_{i}).
+
+`LGBExitModeler` follows similar rules as `LGBSurvivalModeler` for selecting observations for training and forecasting. Except for training, where only observations with an observed exit in the desired time period are used.
+
+.. code-block:: python
+
+	# Obtain the probability of type of exit conditional on exit that period
+	exit_modeler = LGBExitModeler(data=mdata, exit_col = 'outcome')
+	exit_modeler.build_model(parallelize=False)
+	exit_modeler_forecasts = exit_modeler.forecast()
+	
+	# Set MultiIndex and column names
+	exit_modeler_forecasts = exit_modeler_forecasts.set_index('Future outcome', append = True, drop = True)
+	exit_modeler_forecasts = exit_modeler_forecasts.reindex(['Same government type','Change of government type','Dissolution','No exit'], axis=0,level=1)
+	exit_modeler_forecasts.columns = custom_forecast_headers
+	exit_modeler_forecasts
+	```
+	                                                         Jul 2020      Aug 2020  ...      Jul 2040      Aug 2040
+	country-leader            Future outcome                                         ...                            
+	Afghanistan: Ashraf Ghani Same government type       8.578190e-01  7.704733e-01  ...  7.762426e-01  7.535799e-01
+	                          Change of government type  1.419008e-01  2.289023e-01  ...  2.237574e-01  2.464201e-01
+	                          Dissolution                2.801606e-04  6.244381e-04  ...  9.282013e-16  9.760585e-16
+	                          No exit                    1.004800e-15  1.110349e-15  ...  9.282013e-16  9.760585e-16
+	Albania: Rama             Same government type       9.911523e-01  9.692952e-01  ...  7.521312e-01  7.186383e-01
+	                                                          ...           ...  ...           ...           ...
+	Zambia: Lungu             No exit                    5.816850e-16  5.342558e-16  ...  9.282013e-16  9.760585e-16
+	Zimbabwe: Mnangagwa       Same government type       9.176592e-01  8.384861e-01  ...  6.396734e-01  7.186383e-01
+	                          Change of government type  8.204236e-02  1.608740e-01  ...  3.603266e-01  2.813617e-01
+	                          Dissolution                2.984380e-04  6.399155e-04  ...  1.069257e-15  1.018499e-15
+	                          No exit                    1.070355e-15  1.137919e-15  ...  1.069257e-15  1.018499e-15
+	```
+
+##### Cumulative Incidence Function
+
+It is often of interest to obtain the Cumulative Incidence Function (CIF) defined as :math:`Pr(T_i\leq t,D_i=d|X_{i})`.  The CIF is obtained using the forecasts from `LGBSurvivalModeler` and `LGBExitModeler` using the relationship
```

Comment:
Please give an intuitive description of the CIF, such as "The CIF gives the probability of a specific type of exit on or before each future time period."

Reply:
Done